### PR TITLE
Category

### PR DESCRIPTION
**Pull request by alarcher**
Semi-scripted thing, need to double check.

### CODE REVIEW

**Comment by pyhalov** [comment]
@alarcher Do you have interest in resurrecting this?  If not much, perhaps, close PR for now :)

**Comment by Mno-hime** [comment]
@balrog-nona works on another batch of clean-ups and will eventually move everything to suitable categories.

**Comment by alarcher** [comment]
Rebased

File: components/desktop/gnome3/unico/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # Copyright 2016 Alexander Pyhalov
 #
 
-include ../../make-rules/shared-macros.mk
+include ../../../../make-rules/shared-macros.mk
```
Comment (pyhalov) [inline components/desktop/gnome3/unico/Makefile]: Perhaps, just drop unico, theme engines are not supported in fresh GTK3

**Comment by alarcher** [comment]
@pyhalov I have deprecated unico.

**Pull request merged**